rviz_common/CMakeLists.txt

```diff
@@ -22,16 +22,6 @@ else()
   set(SKIP_DISPLAY_TESTS "SKIP_TEST")
 endif()
 
-if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  add_compile_options(-Wall -Wextra -Wpedantic)
```

Comment:
@richmattes I'm not sure we want to remove this... Most ROS 2 packages have `-Wall -Wextra -pedantic` on, and they should probably remain on.

Reply:
Right, I got a little excited - I've restored the extra warnings.